flask_dynamo/manager.py

```diff
@@ -32,137 +78,125 @@ def init_app(self, app):
 
         :param obj app: The Flask application.
         """
-        self.app = app
-        self.init_settings()
-        self.check_settings()
 
-    def init_settings(self):
+        self._init_settings(app)
+        self._check_settings(app)
+
+        app.extensions['dynamo'] = self
+
+        self.tables = DynamoLazyTables(self.connection, app.config['DYNAMO_TABLES'])
+
+    @staticmethod
+    def _init_settings(app):
         """Initialize all of the extension settings."""
-        self.app.config.setdefault('DYNAMO_TABLES', [])
-        self.app.config.setdefault('DYNAMO_ENABLE_LOCAL', environ.get('DYNAMO_ENABLE_LOCAL', False))
-        self.app.config.setdefault('DYNAMO_LOCAL_HOST', environ.get('DYNAMO_LOCAL_HOST'))
-        self.app.config.setdefault('DYNAMO_LOCAL_PORT', environ.get('DYNAMO_LOCAL_PORT'))
-        self.app.config.setdefault('AWS_ACCESS_KEY_ID', environ.get('AWS_ACCESS_KEY_ID'))
-        self.app.config.setdefault('AWS_SECRET_ACCESS_KEY', environ.get('AWS_SECRET_ACCESS_KEY'))
-        self.app.config.setdefault('AWS_REGION', environ.get('AWS_REGION', self.DEFAULT_REGION))
+        app.config.setdefault('DYNAMO_TABLES', [])
+        app.config.setdefault('DYNAMO_ENABLE_LOCAL', environ.get('DYNAMO_ENABLE_LOCAL', False))
+        app.config.setdefault('DYNAMO_LOCAL_HOST', environ.get('DYNAMO_LOCAL_HOST'))
+        app.config.setdefault('DYNAMO_LOCAL_PORT', environ.get('DYNAMO_LOCAL_PORT'))
+        app.config.setdefault('AWS_ACCESS_KEY_ID', environ.get('AWS_ACCESS_KEY_ID'))
+        app.config.setdefault('AWS_SECRET_ACCESS_KEY', environ.get('AWS_SECRET_ACCESS_KEY'))
+        app.config.setdefault('AWS_REGION', environ.get('AWS_REGION', Dynamo.DEFAULT_REGION))
```

Comment:
Hi, thanks for this work. We test it currently and it works fine. I just have a suggestion. We use it in a AWS lambda with zappa. So all AWS environment variable are set automatically and the boto session is already active. Could you add 'AWS_SESSION_TOKEN' in the session_kwargs in order to be able to use current session ?

app.config.setdefault('AWS_SESSION_TOKEN', environ.get('AWS_SESSION_TOKEN'))